### PR TITLE
Updates !

### PR DESCRIPTION
**Pull request by almet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,22 +5,53 @@ Whiskerboard is a status board for websites, services and APIs, like Amazon's [A
 
 It is heavily based on [Stashboard](http://www.stashboard.org/). Unlike Stashboard, it uses vanilla Django, so you aren't stuck using Google App Engine.
 
-Have a look at the demo: [http://whiskerboard.ep.io/](http://whiskerboard.ep.io/).
+Have a look at the demo: [http://status.lolnet.org/](http://status.lolnet.org/).
```
Comment (bsdlp) [inline README.md]: demo url seems to be suspended, can you take a look?

Comment (almet) [inline README.md]: yep it is.I should setup it somewhere else but never found the time to :)

Comment (almet) [inline README.md]: that's back online!

Comment (bsdlp) [inline README.md]: \o/ thanks @ametaireau.

On a side note, if you do want to move it somewhere else, I'd be more than happy to help.

**Comment by bfirsh** [comment]
Thanks @ametaireau.

Could some of the bug fixes get squashed and the docs updated so it no longer represents a branch? It would be good to make the HTTP checking optional too (if you don't want to install Celery etc).

**Comment by almet** [comment]
Did that in another PR. Closing this one.

**Pull request closed without merging**